config/routes.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 Rails.application.routes.draw do
+  mount MagicLamp::Genie, at: '/magic_lamp' if defined?(MagicLamp)
```

Comment:
I hate that the test framework is leaking into the application logic itself here, is there any way we can inject this from the test env instead?